nca/SchemeRunner.py

```diff
@@ -198,12 +198,11 @@ def run_queries(self, query_array):
                 if res != expected:
                     self.warning(f'Unexpected result for query {query_name}: Expected {expected}, got {res}\n', query)
                     self.global_res += 1
-            if 'expectedNotExecuted' in query:
-                expected_not_executed = query['expectedNotExecuted']
-                if not_executed != expected_not_executed:
-                    self.warning(f'{query_name} was not executed {not_executed} times. '
-                                 f'Although, expected to not be executed {expected_not_executed} times')
-                    self.global_res += 1
+            expected_not_executed = query.get('expectedNotExecuted', 0)
+            if not_executed != expected_not_executed:
+                self.warning(f'{query_name} was not executed {not_executed} times. '
+                             f'Although, expected to not be executed {expected_not_executed} times')
```

Comment:
I would consider changing the warning if `expectedNotExecuted` was not specified in the scheme file, to just include the text that describes the query was not executed.

Reply:
done